src/components/Form/Inputs/IFieldProps.tsx

```diff
@@ -1,10 +1,11 @@
 import { FieldState } from 'formstate';
 import { ILayer } from './ILayer';
+import { IDatePickerProps } from '@blueprintjs/datetime';
 
 /**
  * Field Props
  */
-export interface IFieldProps {
+export interface IFieldProps extends IDatePickerProps{
```

Comment:
this is the interface used by all the gsi-ui fields and there are all not date time, please make the Vdatetime and the VDateTimeRange only extends this